client/contracts/bindings/consensus/consensus.go

```diff
@@ -77,7 +62,12 @@ type mUSD struct {
 }
 
 func NewMUSD(contractBackend bind.ContractBackend, chainID *big.Int) (*mUSD, error) {
-	mtoken, err := NewMiningToken(mapMiningTokenToAddr[chainID.Uint64()], contractBackend)
+	addr, err := getAddressFromKNS("miningtoken.kowala", contractBackend)
+	if err != nil {
+		return nil, err
+	}
+
+	mtoken, err := NewMiningToken(addr, contractBackend)
```

Comment:
as soon as we update the addr, this will not work anymore

Reply:
@rgeraldes before you go explain me well where is the best place to set the usage so it can be solved in the future. Do you think this is something that we will need now or can wait?